### PR TITLE
refactored multimedia image

### PR DESCRIPTION
**Pull request by kikoruiz**
Just a few formatting and refactoring changes.

### CODE REVIEW

**Comment by kikoruiz** [comment]
@davecarter @danderu @ruben-martin-lozano please check it.

**Comment by ruben-martin-lozano** [comment]
Ahí va mi dedico :+1:

**Comment by davecarter** [comment]
Due to the slider feature is not developed yet I prefer to get rid of the array mapping  to avoid breaking the layout if more than just one picture is provided.

**Comment by ruben-martin-lozano** [comment]
As we've seen together, it's nice that this component can print N images, because of not all projects using "sui-multimedia" will use the future "sui-slider" component, so we'll leave as it is at this moment. 
But we'll update the readme.md information, in order to explain that this component just prints N images with/out links, and if necessary, we'll implement to print videos, canvas, svg, etc. :)

**Comment by kikoruiz** [comment]
Alright! It makes sense. Updating "readme.md" file ;)

File: src/sui-multimedia/index.jsx [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+      <div className='sui-Multimedia'>
+        {this.props.data.map((media, index) => {
+          return media.link ? (
+            <a href={media.link} title={media.alt} key={index}>
+              <img src={media.src} alt={media.alt} className='sui-Multimedia-image' />
+            </a>
+          ) : <img
+                src={media.src}
+                alt={media.alt}
+                className='sui-Multimedia-image'
```
Comment (danderu) [inline src/sui-multimedia/index.jsx]: shouldn't it be `sui-Multimedia--image`? @davecarter just me asking

Comment (kikoruiz) [inline src/sui-multimedia/index.jsx]: I don't think so. `image` is not a modifier, it's an element within `sui-Multimedia`.

Comment (davecarter) [inline src/sui-multimedia/index.jsx]: You're right @kikoruiz 
`--` is a modificator to indicate status (hightlighted, enabled, disabled...)

**Comment by davecarter** [comment]
:+1:

**Comment by ruben-martin-lozano** [comment]
:+1:

File: src/sui-multimedia/index.jsx (hunk 1 above)
Comment (danderu) [inline src/sui-multimedia/index.jsx]: Ok! I learnt something new! :)

**Pull request merged**